Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.yml

```diff
@@ -0,0 +1,376 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Pulsedive
+  version: -1
+configuration:
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- defaultvalue: Low
+  display: Minimum severity of alerts to fetch
+  name: min_severity
```

Comment:
this parameter unnecessary since it not used in the code

Reply:
removed